### PR TITLE
chore: mantine-fy dashboard alert

### PR DESCRIPTION
**Pull request by joaoviana**
Closes: N/A

### Description:

before
Screenshot 2023-12-14 at 22 14 04

after
Screenshot 2023-12-14 at 22 20 06

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Comment by IrakliJani** [comment]
@joaoviana 

ui nits:

we should probably change the layout of this:

CleanShot 2023-12-15 at 19 31 48@2x

option one: move icon to the left and text align left
option two: center action icons as well

File: packages/frontend/src/pages/Dashboard.tsx [hunk 1]
```diff
@@ -501,24 +500,39 @@ const Dashboard: FC = () => {
...
+                            color="red"
+                            size={50}
+                        />
+                        <Text fw={500} ta="center">
+                            You have unsaved changes to your dashboard! Are you
+                            sure you want to leave without saving?
+                        </Text>
+                    </Group>
+                    <Group position="right">
+                        <Button>Stay</Button>
```
Comment (IrakliJani) [inline packages/frontend/src/pages/Dashboard.tsx:522]: does stay close the modal?

Comment (joaoviana) [reply packages/frontend/src/pages/Dashboard.tsx:522]: 🤦

**Comment by joaoviana** [comment]
@IrakliJani 

Screenshot 2023-12-15 at 16 09 17

fixed the issue with `stay` not closing the modal
changed layout ( a mix of your suggestions)
user can't close on click outside

**Comment by IrakliJani** [comment]
merging

**Pull request merged**